### PR TITLE
Helens branch merge

### LINKED ISSUE #245: Automatic Coding
* **Is your feature request related to a problem? Please describe.**

I'm frustrated when I can't do the code

* **Describe the solution you'd like**

Code to be outputted for what I want automatically

* **Describe alternatives you've considered**

Giving up

* **Additional context**

:( <-- visual of me when I can't code

### PR DESCRIPTION
**Pull request by h-sleight**
## Proposed changes

Merging my training branch into the main dev environment

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [x] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have checked that my changes have not broken any other functionality
- [x] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [ ] All of your unit tests pass
- [x] Your code is clear and easy to understand
- [x] You have documented any new features
- [ ] They can run the code you have written
- [x] Your changes do not break any other functionality

## Who is reviewing this PR?

Tag your reviewer here to ensure they get a notification!

### CODE REVIEW

File: text_file.txt [hunk 1]
```diff
@@ -3,3 +3,7 @@ This is a random text file. You can add and remove things to it to test how git
 
 This is some text.
 I am just normally editing a text file here.
+
+Learning Git.
+
```
Comment (Dipad-imrankhan) [inline text_file.txt:8]: get rid of space

File: text_file.txt [hunk 2]
```diff
@@ -3,3 +3,7 @@ This is a random text file. You can add and remove things to it to test how git
 
 This is some text.
 I am just normally editing a text file here.
+
+Learning Git.
```
Comment (Dipad-imrankhan) [inline text_file.txt:7]: me too

**Review by Dipad-imrankhan — APPROVED** [review]
added comments to new lines

File: text_file.txt [hunk 3]
```diff
@@ -3,3 +3,7 @@ This is a random text file. You can add and remove things to it to test how git
...
+
+imran was here
```
Comment (yooanean) [inline text_file.txt:9]: nice comment

**Review by yooanean — APPROVED** [review]

**Pull request closed without merging**